h/services/bulk_api/lms_stats.py

```diff
@@ -1,6 +1,7 @@
 # pylint:disable=not-callable,use-implicit-booleaness-not-comparison-to-zero,singleton-comparison
```

Comment:
I'd like to change the naming of this and the service modules, for example settle for:

services/bulk/lms.py
views/api/bulk/lms.py
for both.


I'd do that (or other name) in a follow up PR to make this easier to review.